qt/scientific_interfaces/Indirect/FqFitDataPresenter.cpp

```diff
@@ -10,69 +10,23 @@
 
 #include "MantidQtWidgets/Common/SignalBlocker.h"
 
+namespace {
+auto &ads_instance = Mantid::API::AnalysisDataService::Instance();
```

Comment:
Just a note about these types of variable. This `Instance` call is run as soon as the program loads, regardless of whether anything actually happens with this interface. Can we move this to be a reference member of the class so that it's initialized in the constructor? Similarly with any other uses like this?